.github/workflows/deploy_latest_release.yml

```diff
@@ -58,3 +70,43 @@ jobs:
           PROJECT: pwa-kit
           MESSAGE: "tag ${{ github.event.release.tag_name }}"
           FLAGS: --cloud-origin https://cloud-testing.mobify-staging.com -c ~/.mobify --wait
+
+  deploy-demo-site:
+    needs: check-latest-release
+    runs-on: ubuntu-latest
+    if: needs.check-latest-release.outputs.IS_LATEST_RELEASE == 'true'
+    steps:
```

Comment:
nit  deploy-bug-bounty and deploy-demo-site share the same steps. Maybe using a matrix can help to remove duplication:

```
strategy:
    matrix:
      environment:
        - name: bug-bounty
          project_key: retail-react-app-bug-bounty
          target: testing
          project: pwa-kit
        - name: demo-site
          project_key: retail-react-app-demo-site
          target: production
          project: scaffold-pwa
```

Reply:
I've updated the workflow to use a matrix @adamraya

Reply:
Here is a CI run that used the matrix: https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/13801491420